packages/dataviews/src/components/dataviews-view-config/index.tsx

```diff
@@ -551,11 +545,8 @@ function DataviewsViewConfigDropdown( {
 								<SortFieldControl />
 								<SortDirectionControl />
 							</HStack>
-							{ view.type === LAYOUT_GRID && (
-								<DensityPicker
-									density={ density }
-									setDensity={ setDensity }
-								/>
+							{ !! usedLayout?.viewConfigOptions && (
```

Comment:
Consumers can provide default data for the density via `view.layout.density`. That's nice. How could they disable this layout-specific config so it still takes a value (e.g.: `compact`) and users cannot change it?

Reply:
Good question! It feels like we'd need a new API for something like that, no? Maybe something like what [Query Loop does](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/query/utils.js#L207)?

Reply:
Not sure how that translates to a DataView concept, could you ellaborate? :)

Reply:
Maybe it could be by having an extra prop in `view.layout` which could be something like `view.layout.renderControls: ['density']`. Having an empty array there could mean do not render any `viewConfigOptions`. I think we'd need a bit more time to see how `viewConfigOptions` evolves to start exploring a good way to support this.

Reply:
That could work, though we need to figure out a few things:

- sort & order are configured by the Field API: if no field is sortable they won't be available
- pagination & field config (moving but also field visibility) cannot be disabled and are controlled by the view config component
- the view config doesn't have information about the layout-specific view configs (density, preview size)